sml/linear_model/tests/logistic_test.py

```diff
@@ -76,6 +98,25 @@ def proc(x, y, penalty):
             # print("Predict result label: ", result[1])
             print(f"{penalty} ROC Score: {roc_auc_score(y.values, result[0])}")
 
+        # Multi classification
```

Comment:
可以把多分类的测试重新起一个函数哈，，emulation同理～（注意减少代码重复）

Reply:
减少代码重复上，可以把sim  loaddata提取出一个函数，proc可以提取出一个函数，proc这里有个疑问，比如我要传多个静态变量进去应该怎么写，例如：
`def proc(x, y, penalty, class_labels）`
执行的时候,static_argnums和下面的参数不知道该怎么写......,下面的写法是错误的
```
result3 = spsim.sim_jax(sim, proc2, static_argnums=(2,))(
    X.values, y.values.reshape(-1, 1), ，"l2", [0,1,2]
)
```

Reply:
你这个例子里我理解static_argnums=(2,3)?

Reply:
直接传数组会报错：
`ValueError: Non-hashable static arguments are not supported, as this can lead to unexpected cache-misses. Static argument (index 3) of type <class 'list'> for function proc2 is non-hashable.`
打算传"0，1，2"字符串进去再做处理

Reply:
不能放list这种可变对象，可以用tuple

Reply:
或者，也可以直接传一个数字，因为assume了label一定是0,1,2...